test/Microsoft.AspNetCore.Razor.Test/TestFiles/CodeGenerator/Output/AttributeTargetingTagHelpers.cs

```diff
@@ -48,7 +48,7 @@ public AttributeTargetingTagHelpers()
                 await __tagHelperRunner.RunAsync(__tagHelperExecutionContext);
                 if (!__tagHelperExecutionContext.Output.IsContentModified)
                 {
-                    __tagHelperExecutionContext.Output.Content = await __tagHelperExecutionContext.Output.GetChildContentAsync();
+                    await __tagHelperExecutionContext.SetOutputContentAsync();
```

Comment:
In design time we don't ever generate this line because it doesn't effect  the design time experience. Aka, it's unneeded code at design time.